library/ssl_srv.c

```diff
@@ -4416,4 +4432,10 @@ int mbedtls_ssl_handshake_server_step( mbedtls_ssl_context *ssl )
 
     return( ret );
 }
+
+void mbedtls_ssl_conf_respect_client_preference( mbedtls_ssl_config *conf, int enable )
+{
+    conf->respect_cli_pref = enable;
```

Comment:
Possibly we should check that the value of enable is one of the two permitted values here? But most of the other conf functions don't seem to do this, so I'm ok with not doing this.